tools/report-converter/report_converter/converter/output_parser.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+from abc import ABCMeta, abstractmethod
+
+
+def get_next(it):
+    """ Returns the next item from the iterator or return an empty string. """
+    try:
+        return next(it)
+    except StopIteration:
+        return ''
+
+
+class Event(object):
+    """ Represents an event message. """
+
+    def __init__(self, path, line, column, message):
+        self.path = path
+        self.line = line
+        self.column = column
+        self.message = message
+
+    def __eq__(self, other):
+        return self.path == other.path and \
+            self.line == other.line and \
+            self.column == other.column and \
+            self.message == other.message
+
+    def __str__(self):
+        return 'path={0}, line={1}, column={2}, message={3}'.format(
+            self.path, self.line, self.column, self.message)
+
+
+class Message(Event):
+    """ Represents a message with an optional event and fixit messages. """
+
+    def __init__(self, path, line, column, message, checker, events=None,
+                 notes=None, fixits=None):
+        super(Message, self).__init__(path, line, column, message)
+        self.checker = checker
+        self.events = events if events else []
+        self.notes = notes if notes else []
+        self.fixits = fixits if fixits else []
+
+    def __eq__(self, other):
+        return super(Message, self).__eq__(other) and \
+            self.checker == other.checker and \
+            self.events == other.events and \
+            self.notes == other.notes and \
+            self.fixits == other.fixits
+
+    def __str__(self):
+        return '%s, checker=%s, events=%s, notes=%s, fixits=%s' % \
+               (super(Message, self).__str__(), self.checker,
+                [str(event) for event in self.events],
+                [str(note) for note in self.notes],
+                [str(fixit) for fixit in self.fixits])
+
+
+class OutputParser(object):
```

Comment:
It is in the `output_parser` module so `OutputParser` is a bit redundant i think. `output_parser.Parser` or `output.Parser`, `analyzer_output.Parser`?